test/machinegun/config.yaml

```
@@ -3,94 +3,86 @@ erlang:
+        event_sinks:
+            machine:
+                type: machine
+                machine_id: ff/withdrawal/session_v2
+        processor:
+            url: http://fistful-server:8022/v1/stateproc/ff/withdrawal/session_v2
+    ff/w2w_transfer_v1:
+        event_sinks:
+            machine:
+                type: machine
+                machine_id: ff/w2w_transfer_v1
+        processor:
+            url: http://fistful-server:8022/v1/stateproc/ff/w2w_transfer_v1
 
-  # Fistful
-  ff/identity:
-      event_sinks:
-        machine:
-            type: machine
-            machine_id: ff/identity
-      processor:
-          url: http://fistful-server:8022/v1/stateproc/ff/identity
-  ff/wallet_v2:
-      event_sinks:
-        machine:
-            type: machine
-            machine_id: ff/wallet_v2
-      processor:
-          url: http://fistful-server:8022/v1/stateproc/ff/wallet_v2
-  ff/source_v1:
-      event_sinks:
-        machine:
-            type: machine
-            machine_id: ff/source_v1
-      processor:
-          url: http://fistful-server:8022/v1/stateproc/ff/source_v1
-  ff/deposit_v1:
-      event_sinks:
-        machine:
-            type: machine
-            machine_id: ff/deposit_v1
-      processor:
-          url: http://fistful-server:8022/v1/stateproc/ff/deposit_v1
-  ff/destination_v2:
-      event_sinks:
-        machine:
-            type: machine
-            machine_id: ff/destination_v2
-      processor:
-          url: http://fistful-server:8022/v1/stateproc/ff/destination_v2
-  ff/withdrawal_v2:
-      event_sinks:
-        machine:
-            type: machine
-            machine_id: ff/withdrawal_v2
-      processor:
-          url: http://fistful-server:8022/v1/stateproc/ff/withdrawal_v2
-  ff/withdrawal/session_v2:
-      event_sinks:
-        machine:
-            type: machine
-            machine_id: ff/withdrawal/session_v2
-      processor:
-          url: http://fistful-server:8022/v1/stateproc/ff/withdrawal/session_v2
-  ff/w2w_transfer_v1:
-      event_sinks:
-        machine:
-            type: machine
-            machine_id: ff/w2w_transfer_v1
-      processor:
-          url: http://fistful-server:8022/v1/stateproc/ff/w2w_transfer_v1
+    ff/sequence:
+        processor:
+            url: http://fistful-server:8022/v1/stateproc/ff/sequence
+    ff/external_id:
+        processor:
+            url: http://fistful-server:8022/v1/stateproc/ff/external_id
 
-  ff/sequence:
-      processor:
-          url: http://fistful-server:8022/v1/stateproc/ff/sequence
-  ff/external_id:
-      processor:
-          url: http://fistful-server:8022/v1/stateproc/ff/external_id
-
-  # Bender
-  bender_generator:
-    processor:
-      url: http://bender:8022/v1/stateproc/bender_generator
-  bender_sequence:
-    processor:
-      url: http://bender:8022/v1/stateproc/bender_sequence
+    # Bender
+    bender_generator:
+        processor:
+        url: http://bender:8022/v1/stateproc/bender_generator
```

Сломал yaml:
```suggestion
             url: http://bender:8022/v1/stateproc/bender_generator
```

И чуть ниже.

---

Эти неймспейсы нужны для тестов вообще?